File: test/advent2022.test.mjs
import { describe, it } from 'node:test';
import assert from 'node:assert';
import * as dayOne from '../src/01.mjs';

describe('Day 01', { concurrency: true }, () => {
    it('Part One', () => assert.equal(68467, dayOne.partOne()));
    it('Part Two', () => assert.equal(203420, dayOne.partTwo()));
});

File: test/advent2022_test.exs
defmodule Advent2022Test do
  use ExUnit.Case
  doctest Advent2022

  test "partOne" do
    assert 68467 == Advent2022.partOne()
  end

  test "partTwo" do
    assert 203_420 == Advent2022.partTwo()
  end
end
